mungegithub/Makefile

```diff
@@ -69,6 +69,12 @@ secret:
 clean:
 	rm -f mungegithub $(APP)/local.deployment.yaml $(APP)/local.secret.yaml
 
+# pull down current public queue state, and run UI based off that data
+ui-stub:
+	@/bin/bash -c "wget -r -nH -P ./submit-queue/www http://submit-queue.k8s.io/{prs,github-e2e-queue,history,sq-stats,stats,users,health,google-internal-ci,priority-info,merge-info,health}; \
+	pushd ./submit-queue/www; \
+	python -m SimpleHTTPServer;"
```

Comment:
Now this looks cool....  However I'm unaware of the @/bin/bash format. How does that differ from $(shell ....)